src/features/goblins/trader/selling/Selling.tsx

```diff
@@ -28,6 +32,48 @@ export const Selling: React.FC = () => {
     .selling as SellingMachineInterpreter;
   const [machine, send] = useActor(sellingService);
 
+  const { authService } = useContext(AuthProvider.Context);
+  const [authState] = useActor(authService);
+
+  const [jiggerState, setJiggerState] =
+    useState<{ url: string; status: JiggerStatus }>();
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const load = async () => {
+      setIsLoading(true);
+      const check = await loadBanDetails(
+        authState.context.farmId?.toString() as string,
+        authState.context.rawToken as string,
+        authState.context.transactionId as string
+      );
+
+      if (check.verificationUrl) {
+        setJiggerState({
+          url: check.verificationUrl,
+          status: check.botStatus as JiggerStatus,
+        });
+      }
+      setIsLoading(false);
+    };
+
+    load();
+  }, []);
```

Comment:
**suggestion:** This API request is triggering every time the tab is changed between selling and buying.

I think we can consider caching the response, or moving the check up one level into the `TraderModal`.